decentraland/unity-renderer
# Feat: clickable coordinates in chat

## What does this PR change?

Fix #1776 
Detects coordinates in the chat and renders them clickable to teleport to the location
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/clickable-coords
2. Type in chat a message containing coordinates
3. Verify it is clickable and it teleports to the right location

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
